vulnfeeds/cpp/main.go

```diff
@@ -174,11 +181,12 @@ func InScopeRepo(repoURL string) bool {
 }
 
 // Takes an NVD CVE record and outputs an OSV file in the specified directory.
-func CVEToOSV(CVE cves.CVEItem, repo, directory string) {
+func CVEToOSV(CVE cves.CVEItem, directory string) error {
 	CPEs := cves.CPEs(CVE)
 	CPE, err := cves.ParseCPE(CPEs[0])
 	if err != nil {
-		Logger.Fatalf("Can't generate an OSV record for %s without valid CPE data", CVE.CVE.CVEDataMeta.ID)
+		Logger.Warnf("Can't generate an OSV record for %s without valid CPE data", CVE.CVE.CVEDataMeta.ID)
```

Comment:
This is pretty similar to the error that's returned. Should we just have the caller log this?

Reply:
My thinking had been to log as close to the source of the error, not the call site... But I've done as you've requested.

Reply:
In general, the way to pinpoint where things are going wrong is by adding additional context by wrapping (see  
 https://go.dev/blog/go1.13-errors).